graph/src/lib/clientGraph.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Starter LangGraph.js Template
+ * Make this code your own!
+ */
+import {  ToolNode } from '@langchain/langgraph/prebuilt';
+import { ChatOpenAI } from '@langchain/openai';
+import { tokensSearch, bebopRate, EvmKit } from '@agentic-chat/tools';
+import { SYSTEM_PROMPT } from '@agentic-chat/utils';
+import { END, MemorySaver, MessagesAnnotation, START, StateGraph } from '@langchain/langgraph/web';
+import { WalletClient } from 'viem';
+import { AIMessage, SystemMessage } from "@langchain/core/messages";
+import { RunnableLambda } from "@langchain/core/runnables";
+
+// @ts-expect-error TODO: FIXME maybe
+const env = import.meta?.env ? import.meta.env : process.env;
```

Comment:
Should work with the former only, but in reality threw at eval time when trying to do so, so keeping ternary here